app/controllers/partners/users_controller.rb

```diff
@@ -3,30 +3,31 @@ class Partners::UsersController < Partners::BaseController
   before_action :find_account
 
   def index
-    @users = @account.users.page(params[:page])
-    @users = @users.where(open_id: params[:open_id]) if params[:open_id]
+    @users = @account.users.page(permitted_params[:page])
```

Comment:
I don't see `:page` permitted in `#permitted_params`. This will always return `nil` as far as I can tell.

Reply:
@akostadinov Changes done.